variables.tf

```diff
@@ -79,6 +79,458 @@ variable "server_instance_metadata" {
+  default     = "4"
+}
+
+variable "crm_period" {
+  description = "The period of max calls for the CRM  API (in seconds)"
+  default     = "1.2"
+}
+
+variable "crm_disable_polling" {
+  description = "Whether to disable polling for CRM API"
+  default     = "False"
+}
+
+variable "iam_max_calls" {
+  description = "Maximum calls that can be made to IAM API"
+  default     = "90"
+}
+
+variable "iam_period" {
+  description = "The period of max calls for the IAM API (in seconds)"
+  default     = "1.0"
+}
+
+variable "iam_disable_polling" {
+  description = "Whether to disable polling for IAM API"
+  default     = "False"
+}
+variable "logging_max_calls" {
+  description = "Maximum calls that can be made to Logging API"
+  default     = "9"
+}
+
+variable "logging_period" {
+  description = "The period of max calls for the Logging API (in seconds)"
+  default     = "1.0"
+}
+
+variable "logging_disable_polling" {
+  description = "Whether to disable polling for Logging API"
+  default     = "False"
+}
+variable "securitycenter_max_calls" {
+  description = "Maximum calls that can be made to Security Center API"
+  default     = "1"
+}
+
+variable "securitycenter_period" {
+  description = "The period of max calls for the Security Center API (in seconds)"
+  default     = "1.1"
+}
+
+variable "securitycenter_disable_polling" {
+  description = "Whether to disable polling for Security Center API"
+  default     = "False"
+}
+
+variable "servicemanagement_max_calls" {
+  description = "Maximum calls that can be made to Service Management API"
+  default     = "2"
+}
+
+variable "servicemanagement_period" {
+  description = "The period of max calls for the Service Management API (in seconds)"
+  default     = "1.1"
+}
+
+variable "servicemanagement_disable_polling" {
+  description = "Whether to disable polling for Service Management API"
+  default     = "False"
+}
+
+variable "sqladmin_max_calls" {
+  description = "Maximum calls that can be made to SQL Admin API"
+  default     = "1"
+}
+
+variable "sqladmin_period" {
+  description = "The period of max calls for the SQL Admin API (in seconds)"
+  default     = "1.1"
+}
+
+variable "sqladmin_disable_polling" {
+  description = "Whether to disable polling for SQL Admin API"
+  default     = "False"
+}
+
+variable "storage_disable_polling" {
+  description = "Whether to disable polling for Storage API"
+  default     = "False"
+}
+
+variable "cai_api_timeout" {
+  description = "Timeout in seconds to wait for the exportAssets API to return success."
+  default     = "3600"
+}
+
+variable "cai_retention_days" {
+  description = "Number of days to retain inventory data."
+  default     = "-1"
+}
+
+#-------------------------------#
+# Forseti server config scanner #
+#-------------------------------#
+variable "audit_logging_enabled" {
+  description = "Audit Logging scanner enabled."
+  default     = "false"
+}
+
+variable "biquery_enabled" {
```

Comment:
Typo - this should be `bigquery_enabled`, we dropped the first g in `bigquery`

Reply:
Good eye. Changed.